frontend/src/Pages/Teacher/Course/TeacherCourseDetails.jsx

```
@@ -0,0 +1,120 @@
+                    </p>
+
+                    {/* Buttons Section */}
+                    <div className="mt-6 flex gap-4">
+                        <button
+                            onClick={fetchPastSessions}
+                            className={`px-4 py-2 rounded-md transition-all duration-200 ${
+                                showPastSessions
+                                    ? "bg-red-600 hover:bg-red-700"
+                                    : "bg-blue-600 hover:bg-blue-700"
+                            } text-white`}
+                        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)